python/ShapeTools.py

```diff
@@ -137,10 +141,11 @@ def doIndividualModels(self):
                     prop_b = customizer.build(True)
                     if len(sigcoeffs):
                         prop_b.SetName("prop_bin%s_bonly" % b)
-                    sum_b = ROOT.RooRealSumPdf("pdf_bin%s_bonly"       % b,  "", ROOT.RooArgList(prop_b),   ROOT.RooArgList(self.out.var('ONE')), True)
+                    self.objstore[prop_b.GetName()] = prop_b
```

Comment:
Basically just that prop_b isn't constructed directly by us but rather via the customizer. addObj expects to construct the object itself, so we just bypass that here and add it to store